src/data/pokemon/form_change_table_pointers.h

```diff
@@ -1,24 +1,36 @@
 const struct Fusion *const gFusionTablePointers[NUM_SPECIES] =
 {
-#if P_GEN_5_POKEMON == TRUE
+#if P_FAMILY_KYUREM
     [SPECIES_KYUREM] = sKyuremFusionTable,
-    [SPECIES_KYUREM_BLACK] = sKyuremFusionTable,
+#if P_FAMILY_RESHIRAM
     [SPECIES_KYUREM_WHITE] = sKyuremFusionTable,
     [SPECIES_RESHIRAM] = sKyuremFusionTable,
+#endif //P_FAMILY_RESHIRAM
+#if P_FAMILY_ZEKROM
+    [SPECIES_KYUREM_BLACK] = sKyuremFusionTable,
     [SPECIES_ZEKROM] = sKyuremFusionTable,
-#endif
-#if P_GEN_7_POKEMON == TRUE
+#endif //P_FAMILY_ZEKROM
+#endif //P_FAMILY_KYUREM
+#if P_FAMILY_NECROZMA
     [SPECIES_NECROZMA] = sNecrozmaFusionTable,
+#if P_FAMILY_SOLGALEO
     [SPECIES_NECROZMA_DAWN_WINGS] = sNecrozmaFusionTable,
-    [SPECIES_NECROZMA_DUSK_MANE] = sNecrozmaFusionTable,
     [SPECIES_SOLGALEO] = sNecrozmaFusionTable,
+#endif //P_FAMILY_SOLGALEO
+#if P_FAMILY_LUNALA
```

Comment:
These should be `P_FAMILY_COSMOG`. As they are, they're disabled since there's no Solgaleo/Lunala toggles.